indico/htdocs/sass/partials/_tables.scss

```diff
@@ -30,6 +36,27 @@ table.i-table {
         font-weight: bold;
         text-align: left;
         background-color: #EAEAEA;
+
+        &:not(.sorter-false) {
```

Comment:
Or maybe not, `sorter-false` class is used to show that a given column doesn't allow sorting but there are many ways to define column as not sortable. So I think it is better to stick with the above selector.